src/gui/addnewtorrentdialog.cpp

```diff
@@ -788,10 +788,13 @@ void AddNewTorrentDialog::handleRedirectedToMagnet(const QString &url, const QSt
 void AddNewTorrentDialog::handleDownloadFinished(const QString &url, const QString &filePath)
```

Comment:
Do we really need to save the downloaded data to a file? Can't we just not do it instead of worrying about deleting it?

Reply:
PR updated, now it skip the save to file step.